src/components/PlaylistPage/CourseInfo.js

```diff
@@ -0,0 +1,27 @@
+import React, {PropTypes} from 'react';
+import {getCourseIndex} from '../../util';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import styles from './CourseInfo.scss';
+
+const CourseInfo = React.createClass({
+  render() {
+    const courseName = this.props.courseName;
+    const indexfile = getCourseIndex(courseName);
+    const indexfileLength = indexfile.content.toString().length;
+    return (      
+      <div className={this.props.isStudentMode ? styles.containerStudent : styles.containerTeacher}>
+        {indexfileLength != 0 ?
```

Comment:
Husk å bruke `!==` i stedet for `!=` ... men her trenger du vel ikke det?
Holder det ikke med `{indexfile ? ... : ...}`? (evt `{'' + indexfile ? ... : ... }`)